debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -928,6 +931,10 @@ public void shouldExecuteOnConnectStatements() throws Exception {
         waitForStreamingRunning();
 
         SourceRecords actualRecords = consumeRecordsByTopic(6);
```

Comment:
why is this 6 ?

Reply:
Looks like these records are being fetched as part of snapshot. Order is not guaranteed in this case.

Reply:
Got it, I'll add a separate test similar to this test but without ordering constraint.